content/3_Silver/Conclusion.problems.json

```diff
@@ -241,6 +241,19 @@
         "site": "CF"
       }
     },
+    {
+      "uniqueId": "cf-700B",
+      "name": "Connecting Universities",
+      "url": "https://codeforces.com/contest/700/problem/B",
+      "source": "CF",
+      "difficulty": "Normal",
```

Comment:
```suggestion
      "difficulty": "Hard",
```

Reply:
oh yea change the ordering too 💀

Reply:
wow very rare of you to raise the difficulty of a problem